configuration/pih/liquibase/sql/create_functions.sql

```diff
@@ -72,8 +72,9 @@ BEGIN
 	WHERE voided = 0
 	  AND concept_id = _conceptID
 	  AND locale = _locale
```

Comment:
@dmdesimone - it looks like you merged this in, but you never removed the locale constraint here.  You'll need to move the "AND locale = _locale" constraint for this to work as intended.